packages/react-magma-dom/src/components/Dropdown/DropdownExpandableMenuItem.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import styled from '../../theme/styled';
+import { IconProps } from 'react-magma-icons';
+import { AccordionItem, AccordionItemProps } from '../Accordion';
+
+export interface DropdownExpandableMenuItemProps extends AccordionItemProps {
+  /**
+   * Leading icon for the menu item
+   */
+  icon?: React.ReactElement<IconProps>;
+  /**
+   * @internal
+   */
+  index?: number;
+  /**
+   * @internal
+   */
+  isInverse?: boolean;
+  /**
+   * If true, item will be disabled; it will appear dimmed and onClick event (or any other events) will not fire
+   * @default false
+   */
+  disabled?: boolean;
```

Comment:
It looks like `AccordionItemProps` has a `isDisabled` prop. Should we use that instead of `disabled`?

Reply:
I got `disabled` to work the same way, perhaps keeping `disabled` for consistency sake comparative to the rest of Dropdown?

Reply:
Then yeah, we should only use `disabled` for consistency with Dropdown, and should remove all references to `isDisabled`.